### PR TITLE
Update ko.po

### PR DESCRIPTION
**Pull request by JSY1728**
-p.m. 8 : 14 | South Korea
-most translated to Korean.

### CODE REVIEW

**Comment by JSY1728** [comment]
-a.m. 10:33 | South Korea

-I compare to | https://github.com/DawnbrandBots/bastion-bot/pull/131/files | By Ice-Pendragon

File: translations/ko.po [hunk 1]
```diff
@@ -213,368 +220,368 @@ msgstr ""
...
 msgctxt "card-embed"
 msgid "Card Effect"
-msgstr ""
+msgstr "카드 효과"
 
 #: src\locale.ts:46
 msgctxt "command-option"
 msgid "result-language"
-msgstr ""
+msgstr "출력 언어"
```
Comment (kevinlul) [inline translations/ko.po:529]: Although these aren't displayed yet, Discord won't allow spaces for this (see https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-naming)

File: translations/ko.po [hunk 2]
```diff
@@ -213,368 +220,368 @@ msgstr ""
...
 
 #: src\locale.ts:66
 msgctxt "command-option"
 msgid "input-language"
-msgstr ""
+msgstr "입력 언어"
```
Comment (kevinlul) [inline translations/ko.po:534]: Same as above.

**Review by kevinlul — CHANGES_REQUESTED** [review]

File: translations/ko.po (hunk 2 above)
Comment (JSY1728) [reply translations/ko.po:534]: alright. i"ll change that.

Comment (JSY1728) [reply translations/ko.po]: > Same as above.

i changed this part.

#: src\locale.ts:46
msgctxt "command-option"
msgid "result-language"
msgstr ""

#: src\locale.ts:66
msgctxt "command-option"
msgid "input-language"
msgstr ""

Comment (kevinlul) [reply translations/ko.po]: Oh no, the Hangul is fine here, `\p{L}` will match it.
image
It's just the _space_ in the middle that Discord won't like for some reason.

Comment (JSY1728) [reply translations/ko.po]: > Oh no, the Hangul is fine here, `\p{L}` will match it. image It's just the _space_ in the middle that Discord won't like for some reason.

oh i see, how can i fix that?

Comment (kevinlul) [reply translations/ko.po]: If it makes sense in Korean without the space in the middle, then we can use that!

Comment (JSY1728) [reply translations/ko.po]: > If it makes sense in Korean without the space in the middle, then we can use that!

wow. that's surprise!

Comment (JSY1728) [reply translations/ko.po]: -so, like that?

#: src\locale.ts:46
msgctxt "command-option"
msgid "result-language"
msgstr "결과언어"

#: src\locale.ts:66
msgctxt "command-option"
msgid "input-language"
msgstr "출력언어"

Comment (kevinlul) [reply translations/ko.po]: Yes, that would work for Discord.

Comment (JSY1728) [reply translations/ko.po]: > Yes, that would work for Discord.

i see. correct!

Comment (Ice-Pendragon) [reply translations/ko.po]: @JSY1728 But isn't the "input language" "입력 언어", not "출력 언어"?

Comment (JSY1728) [reply translations/ko.po]: > @JSY1728 But isn't the "input language" "입력 언어", not "출력 언어"?

sorry, i fix that.

**Review by kevinlul — APPROVED** [review]
🚀 This is working code-wise! @Ice-Pendragon let me know if this is all good and includes #131. I will separately fix the "Pendulum Scale" localization issues on master once merged.

File: translations/ko.po [hunk 3]
```diff
@@ -213,368 +229,368 @@ msgstr ""
...
 msgctxt "card-embed"
 msgid "Card Effect"
-msgstr ""
+msgstr "카드 효과"
 
 #: src\locale.ts:46
 msgctxt "command-option"
 msgid "result-language"
-msgstr ""
+msgstr "결과언어"
```
Comment (Ice-Pendragon) [inline translations/ko.po]:
```suggestion
msgstr "출력 언어"
```

File: translations/ko.po [hunk 4]
```diff
@@ -213,368 +229,368 @@ msgstr ""
...
 
 #: src\locale.ts:66
 msgctxt "command-option"
 msgid "input-language"
-msgstr ""
+msgstr "입력언어"
```
Comment (Ice-Pendragon) [inline translations/ko.po:543]:
```suggestion
msgstr "입력 언어"
```

File: translations/ko.po [hunk 5]
```diff
@@ -213,368 +229,368 @@ msgstr ""
...
 msgctxt "command-option-description"
 msgid "The language to search in, defaulting to the result language."
-msgstr ""
+msgstr "검색할 언어로, 기본적으로 출력 언어로 설정됩니다."
 
 #: src\commands\art.ts:41
 msgctxt "command-option-description"
 msgid "The password, Konami ID, or name to search for a card."
-msgstr ""
+msgstr "카드를 검색할 카드 코드, 코나미 ID, 또는 이름입니다."
```
Comment (Ice-Pendragon) [inline translations/ko.po:573]:
```suggestion
msgstr "패스워드, 코나미 ID, 또는 카드명으로 카드를 검색할 수 있습니다."
```

File: translations/ko.po [hunk 6]
```diff
@@ -213,368 +229,368 @@ msgstr ""
...
 
 #: src\commands\ygoprodeck.ts:40
 msgctxt "command-option-description"
 msgid "The name or password of the card you're looking for."
-msgstr ""
+msgstr "찾고 있는 카드의 이름 또는 카드 코드입니다."
```
Comment (Ice-Pendragon) [inline translations/ko.po:578]:
```suggestion
msgstr "찾으시는 카드의 이름 또는 패스워드입니다."
```

Comment (JSY1728) [reply translations/ko.po:578]: I'll fix at all.

Comment (JSY1728) [reply translations/ko.po]: I fixed that.

-p.m. 7:18 | South Korea
-request by Ice-Pendragon

**Comment by JSY1728** [comment]
-i finished ko.po

-p.m. 11:32 | South Korea

**Comment by kevinlul** [comment]
All good everyone?

**Comment by Ice-Pendragon** [comment]
https://github.com/DawnbrandBots/bastion-bot/pull/130/files#r937570031
Is this fixed already?

**Comment by JSY1728** [comment]
> https://github.com/DawnbrandBots/bastion-bot/pull/130/files#r937570031
> Is this fixed already?

what's a point?

**Comment by Ice-Pendragon** [comment]
> > https://github.com/DawnbrandBots/bastion-bot/pull/130/files#r937570031
> > Is this fixed already?
> 
> what's a point?

You skipped this one, while applying my fix.

**Comment by JSY1728** [comment]
> > > https://github.com/DawnbrandBots/bastion-bot/pull/130/files#r937570031
> > > Is this fixed already?
> > 
> > 
> > what's a point?
> 
> You skipped this one, while applying my fix.

okay. i'll fix that.

**Comment by JSY1728** [comment]
-p.m. 1:34 | South Korea
I fixed | part : #: src\utils.ts:46

**Review by Ice-Pendragon — APPROVED** [review]

**Review by Ice-Pendragon — APPROVED** [review]

**Review by Ice-Pendragon — APPROVED** [review]

**Comment by Ice-Pendragon** [comment]
Sorry, I didn't know how to "Changes approved".

**Pull request merged**